charts/rancher-monitoring/103.0.0+up40.1.2/templates/rancher-monitoring/hardened.yaml

```diff
@@ -125,4 +125,22 @@ spec:
   - Ingress
   - Egress
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: monitoring-coredns-network-policy
+  namespace: kube-system
+spec:
+  ingress:
+    - ports:
+        - port: 9153
```

Comment:
should this be hard-coded? might be something that should be in the `values.yaml`